Library/Formula/tig.rb

```diff
@@ -43,10 +43,18 @@ class Tig < Formula
   def install
     system "./autogen.sh" if build.head?
     system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{etc}"
-    system "make", "install"
+    system "make", "install", "sysconfdir=#{prefix/"share/tig/examples"}"
```

Comment:
If you just want to install an example configuration you could do:

```ruby
pkgshare.install "/path/to/example/config"
```
After `make install` has executed?